ci/nightly/pipeline.template.yml

```diff
@@ -1588,3 +1588,22 @@ steps:
               composition: ruby
         agents:
           queue: linux-aarch64-small
+
+  - group: ":bazel: Bazel"
+    key: bazel
+    steps:
+      - id: bazel-test-x86_64
+        label: ":bazel: Test (x86_64)"
+        command: bin/ci-builder run stable bazel test //... --remote_cache=https://bazel-remote.dev.materialize.com
```

Comment:
Is this service managed by our cloud team?

Reply:
It is! They added it to `i2` as part of https://github.com/MaterializeInc/i2/pull/1847